modules/nw-metallb-configuring-frr-8ks.adoc

```diff
@@ -0,0 +1,84 @@
+// Module included in the following assemblies:
+//
+// * networking/metallb/metallb-frr-k8s.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-metallb-configuring-frr-k8s_{context}"]
+= Activating integration of MetalLB and FRR-K8s
+
+The following shows how to activate `FRR-K8s` as the backend for `MetalLB`.
+
+.Prerequisites
+
+* You have a cluster installed on bare-metal hardware.
+* You have installed the OpenShift CLI (`oc`).
+* You have logged in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Set the `bgpBackend` field of the `MetalLB` CR to `frr-k8s` as in the following example:
+
++
+[source,yaml]
+----
+apiVersion: metallb.io/v1beta1
+kind: MetalLB
+metadata:
+  name: metallb
+  namespace: metallb-system
+spec:
+  bgpBackend: frr-k8s
+----
+
+[id="nw-metallb-configuring-frr-k8s-configuratons_{context}"]
+= FRR configurations
+
+You can create multiple `FRRConfiguration` CRs to use `FRR` services in `MeltalLB`. 
+`MetalLB` generates an `FRRConfiguration` object then merges this with all other configurations that all users have created.
```

Comment:
nit: Metallb generates an frrConfiguration object that will be merged with.. It's not metallb merging them, but the FRR-K8s daemon.

Reply:
ok. I've rephrased as: `MetalLB` generates an `FRRConfiguration` object which `FRR-K8s` merges with....